src/backend/renderer/mod.rs

```diff
@@ -256,6 +256,7 @@ pub trait ImportMemWl: ImportMem {
     /// with an empty list `&[]`, the renderer is allowed to not update the texture at all.
     fn import_shm_buffer(
         &mut self,
+        dh: &mut DisplayHandle<'_>,
         buffer: &wl_buffer::WlBuffer,
```

Comment:
The fact that this function takes a `wl_buffer` instead of a `ManagedBuffer` makes the whole thing pretty awkward. I think we should find a way to allow `ManagedBuffer` to cover EGL buffers as well.